tests/e2e/test_e2e_flow.py

```
@@ -0,0 +1,121 @@
+            if all_ids:
+                collection.delete(ids=all_ids)
+            yield
+            # Clean up again after test
+            collection = chroma_client.get_collection(CHROMA_COLLECTION)
+            # Get all document IDs
+            all_ids = collection.get()["ids"]
+            # Delete all documents
+            if all_ids:
+                collection.delete(ids=all_ids)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)